src/config/cmConfig.ts

```diff
@@ -1,23 +1,24 @@
 import {ConfigValidator} from 'app/utils/configValidator';
-import {IValidateConfig, Config} from './config';
+import {IValidateConfig, Config, TValidCMClientConfig} from './config';
 import {STATICS} from './statics';
 
 export interface ICMConfig extends IValidateConfig {
-	apiUrl: string | undefined;
-	apiKey: string | undefined;
-	projectId: string | undefined;
-	cluster: string | undefined;
-	baseFilePath: string | undefined;
-	region: string | undefined;
+	apiUrl?: string;
+	apiKey?: string;
```

Comment:
apiKey privalomas.

Reply:
cia api key yra optional todel, nes skaitom is `.env`, tai jei ten nieko nebus - pareina undefined reiksme. bent jau taip constructoriuj vyksta, kad ima is .env, o fallbackui nieko nera padaryta. Bet yra methodas `CMConfig.isValid()` su kuriuo devas gali patikrint ar apiKey, apiUrl ir projectId uzsettinti https://github.com/codemash-io/CodeMash.Js/pull/61/files#diff-705baeaf78545cbdee03e7cafe9fb09a75785ad593c38384b394881217536c54R40